src/DBConnectionCaster.hpp

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+#include "DBConnection.hpp"
+
+template <class To>
```

Comment:
@Morheit , on my teste "typename" looks more informative and don't clash with other usages of "class"

Reply:
@malirod , `class` replaced by `typename`